lib/valueflow.cpp

```diff
@@ -5430,53 +7167,54 @@ void ValueFlow::setValues(TokenList& tokenlist,
         VFA(valueFlowSymbolicInfer(symboldatabase, settings)),
-        VFA(analyzeArrayBool(tokenlist, settings)),
-        VFA(analyzeArrayElement(tokenlist, settings)),
-        VFA(analyzeRightShift(tokenlist, settings)),
-        VFA_CPP(valueFlowCondition(ContainerConditionHandler{}, tokenlist, symboldatabase, errorLogger, settings, skippedFunctions)),
+        VFA(valueFlowArrayBool(tokenlist, settings)),
+        VFA(valueFlowArrayElement(tokenlist, settings)),
+        VFA(valueFlowRightShift(tokenlist, settings)),
+        VFA_CPP(
+            valueFlowCondition(ContainerConditionHandler{}, tokenlist, symboldatabase, errorLogger, settings, skippedFunctions)),
         VFA(valueFlowAfterAssign(tokenlist, symboldatabase, errorLogger, settings, skippedFunctions)),
         VFA_CPP(valueFlowAfterSwap(tokenlist, symboldatabase, errorLogger, settings)),
         VFA(valueFlowCondition(SimpleConditionHandler{}, tokenlist, symboldatabase, errorLogger, settings, skippedFunctions)),
-        VFA(analyzeInferCondition(tokenlist, settings)),
-        VFA(analyzeSwitchVariable(tokenlist, symboldatabase, errorLogger, settings)),
+        VFA(valueFlowInferCondition(tokenlist, settings)),
+        VFA(valueFlowSwitchVariable(tokenlist, symboldatabase, errorLogger, settings)),
         VFA(valueFlowForLoop(tokenlist, symboldatabase, errorLogger, settings)),
-        VFA(analyzeSubFunction(tokenlist, symboldatabase, errorLogger, settings)),
-        VFA(analyzeFunctionReturn(tokenlist, errorLogger, settings)),
+        VFA(valueFlowSubFunction(tokenlist, symboldatabase, errorLogger, settings)),
+        VFA(valueFlowFunctionReturn(tokenlist, errorLogger, settings)),
         VFA(valueFlowLifetime(tokenlist, errorLogger, settings)),
         VFA(valueFlowFunctionDefaultParameter(tokenlist, symboldatabase, errorLogger, settings)),
         VFA(valueFlowUninit(tokenlist, errorLogger, settings)),
         VFA_CPP(valueFlowAfterMove(tokenlist, symboldatabase, errorLogger, settings)),
         VFA_CPP(valueFlowSmartPointer(tokenlist, errorLogger, settings)),
-        VFA_CPP(analyzeIterators(tokenlist, settings)),
+        VFA_CPP(valueFlowIterators(tokenlist, settings)),
         VFA_CPP(
             valueFlowCondition(IteratorConditionHandler{}, tokenlist, symboldatabase, errorLogger, settings, skippedFunctions)),
-        VFA_CPP(analyzeIteratorInfer(tokenlist, settings)),
+        VFA_CPP(valueFlowIteratorInfer(tokenlist, settings)),
         VFA_CPP(valueFlowContainerSize(tokenlist, symboldatabase, errorLogger, settings, skippedFunctions)),
         VFA(valueFlowSafeFunctions(tokenlist, symboldatabase, errorLogger, settings)),
     });
 
     runner.run_once({
         VFA(valueFlowDynamicBufferSize(tokenlist, symboldatabase, errorLogger, settings)),
-        VFA(analyzeDebug(tokenlist, errorLogger, settings)), // TODO: add option to print it after each step/iteration
+        VFA(valueFlowDebug(tokenlist, errorLogger, settings)),
```

Comment:
The change from https://github.com/danmar/cppcheck/commit/0cf639caa8ba951f877f0404c10829a0516c2aff#diff-16a9ec3ff1314646e106eb413234f5614e3a7f527fb82d0a369755bee65a7e31 is lost.

Reply:
You mean the comment?

Reply:
Yes. Nothing to be addressed here. Just to note the differences I spotted during the review.